deploy/jbossas/kit/jboss-wf/org/modeshape/index-provider/lucene/main/module.xml

```diff
@@ -18,10 +18,15 @@
 <module xmlns="urn:jboss:module:1.3" name="org.modeshape.index-provider.lucene">
     <resources>
         <resource-root path="modeshape-lucene-index-provider-${project.version}.jar" />
+        <resource-root path="lucene-analyzers-common-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-core-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-queries-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-queryparser-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-sandbox-${version.org.apache.lucene}.jar" />
+        <resource-root path="lucene-backward-codecs-${version.org.apache.lucene}.jar" />
```

Comment:
IMO it only make sense to have a top-level Lucene libary in WF if more than one module depends on it (or parts of it). Note that WF already has a `org.apache.lucene` module but with a different version.

This type of changes makes clear the intent that this is a private dependency of a module, regardless of what's in WF.

The other reason for this change is that in the case of EAP there's very strict monitoring of the modules which are installed/deployed. Having a top-level `org.apache.lucene` module with a different version than the default one (which is built from sources) will probably create a lot of productization issues.